index.js

```diff
@@ -0,0 +1,7 @@
+import ADBDevice from './lib/adb-device';
+
+function connectToDevice (type) {
```

Comment:
what is the utility of this function? why wouldn't someone just run `new ADBDevice` on their own?